vta/python/vta/top/graphpack.py

```diff
@@ -309,18 +329,30 @@ def graph_pack(expr,
         The bit-width of the weights.
 
     start_name: str, optional
-       Start packing from certain known node.
+       Start packing from certain known node when start_name_idx is None.
 
     stop_name: str, optional
-       Stop packing from certain known node.
+       Stop packing from certain known node when stop_name_idx is None.
+
+    start_name_idx: str, optional
+        When start_name_idx not None, start packing only when node name equal start_name
+        and node idx equal start_name_idx.
+
+    stop_name_idx: str, optional
+        When stop_name_idx not None, stop packing only when node name equal stop_name
+        and node index equal stop_name_idx.
+
+    count_meta:boolean, optional
+        start_name_idx and stop_name_idx count meta or not.
```

Comment:
Can you explain what is the meta here?

Reply:
operator can be relay.expr.Constant which is meta data, default the operator increase logic would not count the meta when count_meta is False , the index number used in graph_pack following the index from  'print(mod.astext(show_meta_data=False))' .

Reply:
Thanks for the clear explanation. I think it would be good to have the explanation in this line of code.